annotations/tekton-annotations/src/main/java/io/dekorate/tekton/decorator/AddJibGradleBuildStepDecorator.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2018 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **/
+
+package io.dekorate.tekton.decorator;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import io.dekorate.DekorateException;
+import io.dekorate.deps.kubernetes.api.model.EnvVarBuilder;
+import io.dekorate.deps.tekton.pipeline.v1beta1.TaskSpecFluent;
+import io.dekorate.utils.Strings;
+
+public class AddJibGradleBuildStepDecorator extends NamedTaskDecorator implements StepDecorator {
+
+  private static final String BUILD_AND_PUSH = "build-and-push";
+  private static final String BUILDER_IMAGE_REF = "$(inputs.params.BUILDER_IMAGE)";
```

Comment:
I assume this syntax if something Tekton specific?

Reply:
Yes